templates/project-ts/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "env": {
```

Comment:
When running Jest tests for the template tests, I'm getting
```
λ npx eslint src/* --fix

C:\Code\snapp-playground\yoni-lint\eslint\src\index.test.ts
  3:1  error  'describe' is not defined  no-undef
  4:3  error  'describe' is not defined  no-undef
  5:5  error  'it' is not defined        no-undef
  6:7  error  'expect' is not defined    no-undef

C:\Code\snapp-playground\yoni-lint\eslint\src\index.ts
  1:22  error  Parsing error: Unexpected token :

✖ 5 problems (5 errors, 0 warnings)
```
I think you can add `"jest": true` under the `env` property and that will fix it.